README.md

```diff
@@ -1,247 +1,260 @@
-	   }
-	 });
+[https://centrifugal.github.io/centrifugo/server/http_api/#history](https://centrifugal.github.io/centrifugo/server/http_api/#hisotry)
 
-	centrifuge.connect();
-
-	// If you using jbuiler use raw render(template_name) and use a global window object to load the user information.
+```ruby
+client.history('chat') 
+
+# {
+#   'result' => {
+#     'publications' => [
+#       {
+#         'data' => {
+#           'text' => 'hello'
+#         },
+#         'uid' => 'BWcn14OTBrqUhTXyjNg0fg'
+#       },
+#       {
+#         'data' => {
+#           'text' => 'hi!'
+#         },
+#         'uid' => 'Ascn14OTBrq14OXyjNg0hg'
+#       }
+#     ]
+#   }
+# }
 ```
 
-If you want to batch sign channels request to avoid multiple HTTP requests, you can use centrifuge JS client to batch channels ```centrifuge.startBatching(); // your code // centrifuge.stopBatching();``` in one request and on the server iterate through channels and subscribe.
+#### Channels
 
-Read more: [JS client Documentation](https://fzambia.gitbooks.io/centrifugal/content/client/api.html)
+Get list of active(with one or more subscribers) channels.
 
-### API request sign
-
-When you use Centrifugo server API you should sign each request to successfully authorize with the server.
+[https://centrifugal.github.io/centrifugo/server/http_api/#channels](https://centrifugal.github.io/centrifugo/server/http_api/#channels)
 
 ```ruby
+client.channels
+
+# {
+#   'result' => {
+#     'channels' => [
+#       'chat'
+#     ]
+#   }
+# }
+```
 
-	# For example this is how you would encode a Publish command
-	commands = {
-		"method": "publish",
-	  	"params": {
-			"channel": "Test",
-		  	"data": { "name": "John Doe" }
-	  	}
-	}
+#### Info
 
-	encoded_data = MultiJson.dump(commands) # Or use to_json
-	sign = client.sign(encoded_data) #Sign the data
+Get running Centrifugo nodes information.
 
-	# Using HTTP party GEM interact with Server API
-	r = HTTParty.post(client.url.to_s, query: {"sign": sign, "data": encoded_data})
-	r.parsed_response
+[https://centrifugal.github.io/centrifugo/server/http_api/#info](https://centrifugal.github.io/centrifugo/server/http_api/#info)
 
+```ruby
+client.info
+
+# {
+#   'result' => {
+#     'nodes' => [
+#       {
+#         'name' => 'Alexanders-MacBook-Pro.local_8000',
+#         'num_channels' => 0,
+#         'num_clients' => 0,
+#         'num_users' => 0,
+#         'uid' => 'f844a2ed-5edf-4815-b83c-271974003db9',
+#         'uptime' => 0,
+#         'version' => ''
+#       }
+#     ]
+#   }
+# }
 ```
 
-```encoded_data``` is a JSON string with your API command/commands. See all available commands in [Server API](https://fzambia.gitbooks.io/centrifugal/content/server/api.html) chapter.
-
-
-## Rails assets
+### Errors
 
-To use Centrifuge js client just add this line to your application.js manifest:
+```ruby
+# Raised when response from Centrifugo contains any error as result of API command execution.
+# 
+Cent::ResponseError
```

Comment:
Can we access underlying code and message of Centrifugo error? An example would definitely help

Reply:
In most cases, it will raise HTTP client error(`Faraday::Error`), which in turn has request/response data inside. I can wrap it with native errors later on.

Reply:
I'll add code an message and a spec for it.